### PR TITLE
update rpc

### PR DESCRIPTION
**Pull request by maxnodereal**
## Description

add mainnet RPC

## Rationale

tell us why we need these changes...

### CODE REVIEW

File: docs/tutorials/native-dapp/cli/file-management/overview.md [hunk 1]
```diff
@@ -26,10 +26,25 @@ When running commands that interact with the greenfield, if there is no config/c
 
 Config file example will set up the necessary RPC address and chain id:
 
-```
-rpcAddr = "https://gnfd-testnet-fullnode-tendermint-us.bnbchain.org:443"
-chainId = "greenfield_5600-1"
-```
+<Tabs
```
Comment (KeefeL) [inline docs/tutorials/native-dapp/cli/file-management/overview.md:29]: I think this is not expected:
image

Comment (KeefeL) [reply docs/tutorials/native-dapp/cli/file-management/overview.md:29]: This doc seems to still describe the usage on Testnet, can we verify it on Mainnet, and rewrite this doc according to mainnet steps?

Comment (maxnodereal) [reply docs/tutorials/native-dapp/cli/file-management/overview.md:29]: Resolved now

File: docs/tutorials/dapp/quick-start.mdx [hunk 2]
```diff
@@ -100,17 +119,39 @@ It turns out that AccountB is not permitted to access the file, which is expecte
...
-```
+<Tabs
+  defaultValue="mainnet"
+  values={[
+    {label: 'Mainnet', value: 'mainnet'},
+    {label: 'Testnet', value: 'testnet'},
+  ]}>
+  <TabItem value="mainnet">
+
+	export RPC_TEST=https://greenfield-chain.bnbchain.org:443
```
Comment (KeefeL) [inline docs/tutorials/dapp/quick-start.mdx:130]: I think this is not correct, we should use the BSC mainnet rpc here, not Greenfield Mainnet RPC

Comment (maxnodereal) [reply docs/tutorials/dapp/quick-start.mdx:130]: i think this is for `gnfd-cmd` config, so we should use gnfd RPC

Comment (KeefeL) [reply docs/tutorials/dapp/quick-start.mdx]: Nope, this is for deploying smart contracts on BSC side, so we need to use bsc mainnet rpc here.

**Review by KeefeL — APPROVED** [review]

**Pull request merged**